justfile

```diff
@@ -73,8 +73,12 @@ env:
     cp api/env.template api/.env
     cp ingestion_server/env.template ingestion_server/.env
 
+# Ensure all services are up and running
+@_api-up: up wait-for-ing wait-for-web
+    exit 0
```

Comment:
This seems kind of like a weird way to write this recipe. Wouldn't this be a little more expected/natural?

```suggestion
# Ensure all services are up and running
@_api-up:
    just up
    just wait-for-ing
    just wait-for-web
```

Reply:
I don't know who wrote the recipe like that but it's been fixed in 670dd8a.